csrankings-c.csv

```diff
@@ -488,6 +488,7 @@ Chengjie Sun,Harbin Institute of Technology,http://homepage.hit.edu.cn/sunchengj
 Chengjun Liu,NJIT,http://cs.njit.edu/liu,TvBcLGMAAAAJ
 Chengkai Li,University of Texas at Arlington,http://ranger.uta.edu/~cli,v8ZQDf8AAAAJ
 Chenglin Miao,University of Georgia,https://clmiao.github.io,QFs2tZoAAAAJ
+Chengmo Yang,University of Delaware,https://www.eecis.udel.edu/~chengmo/,d8KWMt8AAAAJ

```

Comment:
No note of CS cross-appointment - please clarify.

Reply:
Hello Emery, here is a link to the CS joint appointment: https://www.cis.udel.edu/people/faculty-directory/#joint
Please let me know if this resolves the issue. Thanks!